parsers/GT.py

```diff
@@ -0,0 +1,137 @@
+# The arrow library is used to handle datetimes
+import arrow
+# The request library is used to fetch content through HTTP
+import requests
+# The pandas library is used to manipulate real time data
+import pandas as pd
+
+tz_gt = 'America/Guatemala'
+
+MAP_GENERATION = {
+    'coal': 'Turbina de Vapor',
+    'gas': 'Turbina de Gas',
+    'hydro': u'Hidroeléctrica',
+    'oil': 'Motor Reciprocante',
+    'solar': 'Fotovoltaica',
+    'wind': u'Eólico',
+    'geothermal': u'Geotérmica',
+}
+
+
+def fetch_hourly_production(country_code, obj, hour, date):
+    
+    #output frame
+    data = {
+        'countryCode': country_code,
+        'production': {},
+        'storage': {},
+        'source': 'www.amm.org.gt',
+    }
+    
+    #Fill datetime variable
+    data['datetime'] = arrow.get(date, 'DD/MM/YYYY').replace(hour=hour).datetime   
```

Comment:
Don't think so because timezone is already taken into account in the input itself. The hourly functions are used for both:
-`formatted_date = arrow.now(tz=tz_gt).format('DD/MM/YYYY')`
-`past_formatted_date = arrow.get(formatted_date, 'DD/MM/YYYY').shift(days=-1).format('DD/MM/YYYY')`